app/Http/Controllers/NotificationController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Controller\ControllerPosition;
+use App\Models\Notification\Notification;
+use Illuminate\Http\JsonResponse;
+
+class NotificationController extends BaseController
+{
+    public function getActiveNotifications() : JsonResponse
+    {
+        $notifications = Notification::active()
+            ->orderBy('valid_from', 'desc')
+            ->with('controllers')
+            ->get()
+            ->each(function (Notification $notification) {
+
+                $notification->controllers->each(function (ControllerPosition $controllerPosition) {
+                    $controllerPosition->setHidden([
+                        'pivot', 'id', 'frequency', 'created_at', 'updated_at'
+                    ]);
+                });
+
+                $notification->setHidden([
+                    'id', 'disabled_at', 'created_at', 'updated_at'
```

Comment:
Do we want to set `id` as hidden? We appear to be using it later on to mark the notification as read?

Reply:
8bffce3